klee/include/klee/util/Assignment.h

```diff
@@ -74,6 +74,10 @@ class Assignment {
                                 bool _allowFreeValues = false) {
         return AssignmentPtr(new Assignment(objects, values, _allowFreeValues));
     }
+
+    static AssignmentPtr create(const AssignmentPtr &a) {
+        return AssignmentPtr(new Assignment(*a));
+    }
```

Comment:
At first I was thinking if there's any data members that I need to manually clone, but it turns out the default copy constructor can be used directly.

https://github.com/S2E/s2e/blob/bf2b660f06955a51b67df25546784d51f2c9770c/libs2eplugins/src/s2e/Plugins/VulnerabilityAnalysis/PovGenerator.cpp#L199